docs/en/sql-reference/00-sql-reference/30-table-engines/03-icebrg.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Apache Iceberg Engine
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.262"/>
+
+Databend's Apache Iceberg engine allows you to seamlessly query and analyze data in Apache Iceberg tables stored on S3. When you create a table with the Apache Iceberg engine in Databend, you specify a location where the data files of an Apache Iceberg table are stored. This setup allows you to gain direct access to the table and perform queries seamlessly from within Databend.
```

Comment:
not only S3, but also any storage we support.

Reply:
Updated to "...storage in your object storage".